CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## v0.9.0-rc28 [unreleased]
 
+### Bugfixes
+- [#2374](https://github.com/influxdb/influxdb/issues/2374): Two different panics during SELECT percentile
```

Comment:
Link to PR is preferable, but either works. If the PR mentions the issue then they'll be linked together either way.